app/controllers/chargeback_controller.rb

```diff
@@ -655,7 +655,9 @@ def cb_rate_set_record_vars
       rate_tiers = []
       @edit[:new][:tiers][detail_index].each do |tier|
         rate_tier = tier[:id] ? ChargebackTier.find(tier[:id]) : ChargebackTier.new
+        if tier[:start].blank? then tier[:start] = "Infinity" end
```

Comment:
this is shorter:
`tier[:start] = Float::INFINITY if tier[:start].blank?`

and can we use just Float::INFINITY also ? What do you think ?

I am not sure where is place that string "Infinity" is converted to `Float::INFINITY` but noticed a issue that when are manually typing "Infinity" then validation message is displayed and this PR is fixing this case :+1: